app/routes/surveys/components/surveys.css

```diff
@@ -406,3 +402,13 @@
     padding-bottom: 15px;
   }
 }
+
+.copied {
+  border-color: var(--color-green-3);
+  color: var(--color-green-3);
+  outline-color: var(--color-green-3) !important;
+
+  &:hover {
+    border-color: var(--color-green-3) !important;
+  }
```

Comment:
Why are these `!important`?

Is it impossible to to fix the order so these are prioritized?

Reply:
hmm I forgot about these ... I'll look into it 🤔